crates/cli/tests/cli_run.rs

```diff
@@ -976,13 +976,13 @@ mod cli_run {
 
             // TODO fix QuicksortApp and then remove this!
             match roc_filename {
-                "QuicksortApp.roc" => {
+                "quicksortApp.roc" => {
```

Comment:
File naming stuff changed a while ago but we were missing it because the benchmarks test was not checking the exit code.